README.md

```diff
@@ -0,0 +1,34 @@
+# ftrack web widget
+
+This small library encapsulates some logic which can be used to build custom dashboard widgets for ftrack.
+
+## Installation
+
+We recommend installing and bundling with NPM:
+
+```javascript
+npm install @ftrack/web-widget
+```
+
+## Usage
+
+To use the library, define two functions `onWidgetLoad` and `onWidgetUpdate`
+and initialize the library once the document is ready:
+
+```javascript
+import ftrackWidget from "@ftrack/web-widget";
```

Comment:
It doesn't look like ftrackWidget.js has a default export. I might be misstaken, but think you'll need to change this line to:

```
import * as ftrackWidget from "@ftrack/web-widget";
```

or 

```
import { initialize } from "@ftrack/web-widget";
```

Reply:
nice catch, i changed to the second option